iceoryx_hoofs/include/iceoryx_hoofs/memory/relative_pointer.inl

```diff
@@ -239,11 +230,71 @@ inline typename RelativePointer<T>::ptr_t RelativePointer<T>::computeRawPtr() co
     return getPtr(segment_id_t{m_id}, m_offset);
 }
 
+// AXIVION Next Construct AutosarC++19_03-A15.5.3, AutosarC++19_03-A15.4.2, FaultDetection-NoexceptViolations : False
+// positive, std::terminate is not called in the c'tor of PointerRepository and noexcept-specification is not violated
 inline PointerRepository<segment_id_underlying_t, UntypedRelativePointer::ptr_t>& getRepository() noexcept
 {
+    // AXIVION Next Construct AutosarC++19_03-A3.3.2 : PointerRepository can't be constexpr, usage of the static
```

Comment:
This rule appears not to work correctly. It will request making `static` variables `const` (or even `constexpr`) when the intention is that the variable is just that, variable i.e. mutable.

This leads to lots of suppression comments for static variables which is not useful. Having static variables has its use (e.g. lazy singleton as here).

I suggest disabling the rule.

Reply:
@MatthiasKillat I agree, let's monitor this a bit more and then come a decision.

cc @elBoberido